app/controllers/forest_liana/actions_controller.rb

```diff
@@ -0,0 +1,14 @@
+module ForestLiana
+  class ActionsController < ForestLiana::ApplicationController
```

Comment:
This change needs to change custom actions implémentation on the client app side (see https://github.com/ForestAdmin/forest-movies/pull/2/files). Thoughts?